src/mlpack/tests/radical_test.cpp

```diff
@@ -21,8 +21,12 @@ using namespace arma;
 TEST_CASE("Radical_Test_Radical3D", "[RadicalTest]")
 {
   mat matX;
-  data::Load("data_3d_mixed.txt", matX);
-
+  
+  if (!data::Load("data_3d_mixed.txt", matX)){
+    
+  FAIL("Unable to load data_3d_mixed.txt!");
+  }
```

Comment:
No need for the `{}` if it's just a single line, also can you use the same style as used here: https://github.com/mlpack/mlpack/pull/2716